packages/client/hmi-client/src/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss.vue

```diff
@@ -179,11 +177,144 @@ const modelConfigId = computed<string | undefined>(() => props.node.inputs[0]?.v
 const datasetId = computed<string | undefined>(() => props.node.inputs[1]?.value?.[0]);
 const currentDatasetFileName = ref<string>();
 const simulationIds = computed<any | undefined>(() => props.node.outputs[0]?.value);
+
 const runResults = ref<RunResults>({});
-const parameterResult = ref<{ [index: string]: any }[]>();
+const completedRunId = ref<string>();
+
+const showSpinner = ref(false);
+const progress = ref({ status: ProgressState.RETRIEVING, value: 0 });
+
 const mapping = ref<CalibrateMap[]>(props.node.state.mapping);
 
-// Tom TODO: Make this generic... its copy paste from node.
+// EXTRA section
+const numSamples = ref(100);
+const numIterations = ref(100);
+const method = ref('dopri5');
+const ciemssMethodOptions = ref(['dopri5', 'euler']);
+
+const poller = new Poller();
+
+const disableRunButton = computed(
+	() =>
+		!currentDatasetFileName.value ||
+		!modelConfig.value ||
+		!csvAsset.value ||
+		!modelConfigId.value ||
+		!datasetId.value
+);
+
+onMounted(() => {
+	const runIds = querySimulationInProgress(props.node);
+	if (runIds.length > 0) {
+		getStatus(runIds[0]);
+	}
+});
+
+onUnmounted(() => {
+	poller.stop();
```

Comment:
as we are defining a poller in the drill down and the node.
And also stopping them on unmount
We are getting errors when opening and closing the node.

![image](https://github.com/DARPA-ASKEM/terarium/assets/17088680/e46aeb4c-66f3-4689-884d-38aa2053c733)

Reply:
leaving this as it is for now as discussed on slack. will address in a future PR after figuring out how to handle polling if drilldown is unmounted.